crowbar_framework/app/models/crowbar_service.rb

```diff
@@ -32,6 +32,21 @@ def role_constraints
     end
   end
 
+  def shutdown_openstack_at_nodes
+    proposal = ProposalObject.find_proposal('crowbar', 'default')
```

Comment:
Lint/UselessAssignment: Useless assignment to variable - `proposal`. (https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)<br>Style/StringLiterals: Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping. (https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)